labs/lab10/skel/.gitignore

```diff
@@ -1,6 +0,0 @@
-*.exe
-Debug/
-*.user
-*.ncb
-*.txt
```

Comment:
My bad, these lines should be in the `.gitignore`. I only meant to say the last line `tema5` should be removed.